src/Cake.XdtTransform/Cake.XdtTransform.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
```

Comment:
Only target `netstandard2.0` otherwise this will cause problems for Cake. See similar issue [here](https://github.com/cake-build/cake/commit/716feb9418923c69c17ce2e7d448af97fee0e63d).

Reply:
@gitfool Thanks for reviewing!

In this particular case, targeting .NET 5 is totally fine because this is a Cake Addin. The issue [you pointed out](https://github.com/cake-build/cake/commit/716feb9418923c69c17ce2e7d448af97fee0e63d) about having to target `netstandard2.0` **only applies for Cake Modules** because of a limitation on how modules are loaded at the moment (which we'll hopefully fix soon).

- [Recommendations for Addins](https://cakebuild.net/docs/extending/addins/best-practices#net-target-version)
- [Recommendations for Modules](https://cakebuild.net/docs/extending/modules/best-practices#net-target-version)

Reply:
This has been documented too: https://cakebuild.net/docs/extending/modules/best-practices#net-target-version

Reply:
@augustoproiete just saw your comment now. Good to know, thanks!

Reply:
Still, if you don't need any features of `net5.0` then `netstandard2.0` only should be preferred.

Reply:
In theory yes, and different people have different opinions about this so not an exact science, but I personally would recommend addins to always target the highest version that the Cake runners target, especially if you have external dependencies, due to how the Cake runner performs assembly resolution which varies based on the .NET SDK the Cake  runner is running on.

For example, one of my addins didn't target `net5.0` and depends on `System.Runtime.InteropServices.RuntimeInformation` which the Cake runner when running on .NET 5 targets a different version, so my logs were full of warnings saying that Cake was not able to load the assembly that was needed because the Cake runner itself had a newer version (net5.0) loaded already, which adds a lot of noise. Updating my addin (and dependencies) to target `net5.0` removed all the warnings.